src/components/RadioButtonGroup/radioButtonItem.js

```diff
@@ -37,6 +38,10 @@ export default class RadioButtonItem extends Component {
             >
                 <StyledInput
                     type="radio"
+                    ref={this.inputRef}
+                    onClick={() => {
+                        this.inputRef.current.focus();
+                    }}
```

Comment:
same put this in a class method and also check for call this only when safari and mozilla